src/routes/(protected)/calendar/+page.svelte

```
@@ -48,24 +53,111 @@
+            return hasTmdb ? `/details/media/${item.tmdb_id}/movie` : null;
+        }
+
+        if (!hasTmdb && !hasTvdb) {
+            if (import.meta.env.DEV) {
+                console.warn(`Calendar: missing tmdb_id and tvdb_id for "${item.show_title}"`);
+            }
+            return null;
+        }
+
+        const mediaId = hasTmdb ? `${item.tmdb_id}` : `${item.tvdb_id}`;
+        const params = new URLSearchParams();
```

ESLint

[error] Found a mutable instance of the built-in URLSearchParams class. Use SvelteURLSearchParams instead.

(svelte/prefer-svelte-reactivity)